guides/v2.1/javascript-dev-guide/widgets/jquery-widgets-about.md

```diff
@@ -21,6 +21,7 @@ This guide discusses the following widgets:
 -   [Menu widget]
 -   [Modal widget]
 -   [Navigation widget]
+-   [PasswordStrengthIndicator widget]
```

Comment:
Please remove changes to the 2.1 version. You should make these in the 2.2 version.